config/database.py

```diff
@@ -47,3 +60,16 @@
 
 DB = DatabaseManager(DATABASES)
 Model.set_connection_resolver(DB)
+
+
+logger = logging.getLogger('orator.connection.queries')
+logger.setLevel(logging.DEBUG )
+
+formatter = logging.Formatter(
+    'It took %(elapsed_time)sms to execute the query %(query)s'
+)
+
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+
+logger.addHandler(handler)
```

Comment:
What is this addition?

Reply:
I'm not sure but they are in masonite repo.
https://github.com/MasoniteFramework/masonite/blob/8f7a289e1e918b58c1115d669bea8fbe9b8a8a68/config/database.py#L55

Reply:
🤔 i'm not sure why i added these .... lol

Reply:
is it related to logger driver?

Reply:
no it's suppose to show the queries in the terminal (so you can optimize) but i thought thats what the `DB_LOG` was doing .. lol

Reply:
not sure why i added this extra code

Reply:
we can keep it for now and remove it later.

Reply:
?